src/engraving/dom/textbase.h

```diff
@@ -526,13 +526,13 @@ class TextBase : public EngravingItem
 
     Align m_align;
 
-    FrameType m_frameType = FrameType::NO_FRAME;
+    BorderType m_borderType = BorderType::NO_BORDER;
     double m_textLineSpacing = 1.0;
     Color m_bgColor;
-    Color m_frameColor;
-    Spatium m_frameWidth;
+    Color m_borderColor;
+    Spatium m_borderWidth;
     Spatium m_paddingWidth;
-    int m_frameRound = 0;
+    int m_borderRound = 0;
```

Comment:
It might be nice to use the names suggested at https://github.com/musescore/MuseScore/issues/15534#issuecomment-2475025762